openid-4-verifiable-presentations-1_0.md

```diff
@@ -261,6 +261,9 @@ This specification defines the following new parameters:
 
 A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter.
 
+`request_uri_method`: 
+: OPTIONAL. A string determining the HTTP method to be used with the `request_uri` passed in the same request. This specification defines two values for this parameter: `GET` and `POST`. `GET` is the mode as defined by in [@RFC9101], where the Wallet sends a GET request to fetch a request object. `POST` is a new mode defined in this specification where the Wallet requests the creation of a fresh request object as defined in [@RFC9101] by sending a POST request to the request URI. The details of this request are defined in (#request_uri_method_post). `request_uri_mode` MUST only be present if the request also contains a `request_uri` parameter. It SHOULD be sent in a signed authorization request. If the parameter is not present, the Wallet MUST process the `request_uri` as defined in [@RFC9101]. Wallets not supporting the new method "POST" will send a GET request to the request URI (default behavior as defined in [@RFC9101]).
+
```

Comment:
In general I'd prefer `request_uri_mode` since it indicates more than just the HTTP verb is changed, since we are allowing to send some additional parameters in the request.

Reply:
if related to http methods it would be `request_uri_http_methods_supported` which its value type would be a JSON Array

Reply:
my previous comment was related to the Verifier's metadata, I forget to mention it!